env/work/srw_python/SRWLIB_ExampleViewDataFile.py

```diff
@@ -17,20 +18,27 @@
     p.add_option('-y', '--y', dest='y', metavar='NUMBER', type='float', default=0, help='vertical position')
     p.add_option('-l', '--readlab', dest='readlab', metavar='NUMBER', type='int', nargs=0, default=0, help='read labels from the file header (1) or not (0)')
     p.add_option('-j', '--joined', dest='joined', metavar='NUMBER', type='int', nargs=0, default=0, help='place different graphs jointly into one figure (1) or into separate figures (0)')
-    p.add_option('-t', '--traj-report', dest='traj_report', metavar='NUMBER', action='store_true', default=0, help='plot trajectory report') #MR29072016
-    p.add_option('-a', '--traj-axis', dest='traj_axis', metavar='STR', default='x', help='trajectory coordinate ("x" or "y")') #MR29072016
+    p.add_option('-m', '--multicolumn-data', dest='multicolumn_data', metavar='NUMBER', action='store_true', default=0, help='visualize multicolumn data') #MR31102017
+    p.add_option('--column-x', dest='column_x', metavar='STR', default=None, help='column for horizontal axis') #MR31102017
+    p.add_option('--column-y', dest='column_y', metavar='STR', default=None, help='column for vertical axis') #MR31102017
```

Comment:
Please make these option names shorter, and show their "hierarchical dependence", e.g. '--column-x' could be changed to '--m_colx'. The 'm' prefix in '--m_colx' indicates what general option this one refers to. This is how the option names are organized in "virtual beamline" (srwl_bl.py) module.

Reply:
I'll try to rename these options myself as described above, in order to test your "multicolumn" update in uti_plot* modules.

Reply:
Thank you! Please let me know if there are any issues with the visualization and if the underlying functions' arguments are changed. I'll need to update the corresponding Sirepo code.